puppet/manifests/eclipse.pp

```diff
@@ -51,5 +51,10 @@
 		require => Exec['unpack eclipse']
 	}
 
+	exec { 'install Ansi Console plugin':
+		command => '/home/dev/Tools/eclipse/eclipse -application org.eclipse.equinox.p2.director -nosplash -repository http://www.mihai-nita.net/eclipse -installIUs net.mihai-nita.ansicon.feature.group',
+		require => Exec['unpack eclipse']
+	}
```

Comment:
This `exec` has very little to do with the backing up and restoring of databases...
Why is it part of this pull request?